File: Python/A_Project/B_Snake_and_Food/FoodItemFactory/food_item.py
class FoodItem:
    def __init__(self, row: int, col: int):
        self.row = row
        self.col = col
        self.points = 0

    def get_row(self) -> int:
        return self.row

    def get_col(self) -> int:
        return self.col

    def get_points(self) -> int:
        return self.points

File: Ruby/A_Project/B_Snake_And_Food/FoodItemFactory/food_item.rb
# frozen_string_literal: true

class FoodItem
  attr_reader :row, :col, :points

  def initialize(row, col)
    @row = row
    @col = col
    @points = 0
  end

  def get_row
    @row
  end

  def get_col
    @col
  end

  def get_points
    @points
  end
end
